dom0/sd-dom0-qvm-rpc.sls

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+##
+# Explicitly deny as a catch-all for SecureDrop workstation provisioned VMs.
+# All SecureDrop-workstation provisioned VMS should have the sd-workstation tag.
+# To be both be mindful of developers using the workstation and ensure
+# RPC policies are not too permissive, this should be the first action
+# performed by the install. All other provisioning steps will prepend to this
+# list grants.
+##
+
+dom0-rpc-qubes.ClipboardPaste:
+  file.prepend:
```

Comment:
Change `file.prepend` to `file.blockreplace` throughout; see the `OpenInVM` config task for an example.